tensorflow/core/common_runtime/gpu/gpu_process_state.cc

```diff
@@ -174,37 +174,37 @@ std::unique_ptr<SharedCounter> GPUProcessState::ReleaseGPUAllocatorCounter(
   return std::move(allocator_parts.counter);
 #else
   return nullptr;
-#endif
+#endif  // GOOGLE_CUDA || TENSORFLOW_USE_ROCM
 }
 
-Allocator* GPUProcessState::GetCUDAHostAllocator(int numa_node) {
+Allocator* GPUProcessState::GetGPUHostAllocator(int numa_node) {
   CHECK(process_state_);
   if (!HasGPUDevice() ||
-      !process_state_->ProcessState::FLAGS_brain_mem_reg_cuda_dma) {
+      !process_state_->ProcessState::FLAGS_brain_mem_reg_gpu_dma) {
     return process_state_->GetCPUAllocator(numa_node);
   }
   if (numa_node == port::kNUMANoAffinity) {
     numa_node = 0;
   }
   {
-    // Here we optimize the most common use case where cuda_host_allocators_
-    // and cuda_al_ have already been populated and since we're only reading
+    // Here we optimize the most common use case where gpu_host_allocators_
+    // and gpu_host_al_ have already been populated and since we're only reading
```

Comment:
I think this should be removed rather than changed.

Reply:
done